sympy/functions/elementary/complexes.py

```diff
@@ -38,9 +38,34 @@ class re(Function):
     0
     >>> re(im(x) + x*I + 2)
     2
+    >>> re(5 + I + 2)
+    7
+    >>> re(10 + 20*I).as_real_imag()
+    (10, 0)
+    >>> re(10).as_real_imag()
+    (10, 0)
+    >>> re(40*I).as_real_imag()
+    (0, 0)
+    >>> re(x+y*I).as_real_imag()
```

Comment:
I would remove the as_real_imag from the examples here. This should just show what `re` returns. Also, some of these show basically the same thing as the examples above, so one of the two should be removed.

Reply:
Done.